forge-gui/res/cardsfolder/upcoming/bite_down_on_crime.txt

```diff
@@ -0,0 +1,10 @@
+Name:Bite Down on Crime
+ManaCost:3 G
+Types:Sorcery
+S:Mode$ OptionalCost | EffectZone$ All | ValidCard$ Card.Self | ValidSA$ Spell | Cost$ CollectEvidence<6> | Description$ As an additional cost to cast this spell, you may collect evidence 6. This spell costs {2} less to cast if evidence was collected. (To collect evidence 6, exile cards with total mana value 6 or greater from your graveyard.)
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ 2 | EffectZone$ All | Relative$ True | CheckSVar$ Z | Description$ This spell costs {2} less to cast if evidence was collected. (To collect evidence 6, exile cards with total mana value 6 or greater from your graveyard.)
```

Comment:
are you aware it doesn't work?

Reply:
Yeah, I tried different ways to check if collect evidence was paid. Any suggestions what would work here ?

Reply:
Try `Count$OptionalGenericCostPaid.2.0`

it's not perfect but probably ok

Reply:
Thanks, working now !